adhocracy4/maps/widgets.py

```diff
@@ -71,14 +75,23 @@ def render(self, name, value, attrs, renderer=None):
         if hasattr(settings, "A4_OPENMAPTILES_TOKEN"):
             omt_token = settings.A4_OPENMAPTILES_TOKEN
 
+        point = value
+        if isinstance(point, Point):
+            feature = OrderedDict()
```

Comment:
without this the map wouldn't render any more in the dashboard as the point wouldn't be in geojson